packages/lit-dev-content/samples/tutorials/CONTRIBUTING.md

```diff
@@ -0,0 +1,177 @@
+# Authoring a lit.dev Tutorial
+
+Lit tutorials are a way to provide a guided, interactive learning experience to Lit users. Under the hood it leverages playground elements as well as our markdown renderer.
+
+## Steps for Authoring
+
+<details>
+  <summary>Create a directory in <code>/packages/lit-dev-content/tutorials/</code></summary>
+
+This directory name is also going to be a part of the URL e.g. `https://lit.dev/tutorials/tutorial-name`
+</details>
+
+<details>
+  <summary>In that directory create a <code>tutorial.json</code></summary>
+
+See format in following section
```

Comment:
This is kind of dumb maybe but the format looks kind of funky with a summary followed by an outdented details sentence. Maybe convert each of these single steps into single-step bulleted lists, so they match the format of the two-step ones?

This would be a little tricky for the last `<detail>` item which includes a code sample, though. You'd need to indent that as well.

```suggestion
* See format in following section.
```